internal/testrunner/runners/system/runner.go

```diff
@@ -533,6 +616,14 @@ func (r *runner) runTest(config *testConfig, ctxt servicedeployer.ServiceContext
 		return result.WithError(fmt.Errorf("%s", result.FailureMsg))
 	}
 
+	syntheticEnabled, err := r.isSyntheticsEnabled(dataStream, componentTemplatePackage)
+	if err != nil {
+		return result.WithError(err)
```

Comment:
Nit. Add some context on this error.
```suggestion
		return result.WithError(fmt.Errorf("failed to check if synthetic source is enabled", err))
```